mpg321: add page (#4081)
tldr-pages/tldr: pages/common/mpg321.md

# mpg321

> High Performance MPEG 1.0/2.0/2.5 Audio Player for Layer 1, 2, and 3.

> > mpg321 was written to be a drop-in replacement for the (by-then) non-free mpg123 player.

Reviewer: Is this line needed? Feels like it isn't really required.

> - Play an audio source (on repeat) exactly N=3 times:

Reviewer: ```suggestion
- Play an audio source 3 times on repeat:
```

> `mpg321 -l {{3}} {{ file(s) | URL }}`

Reviewer: `{{ file(s) | URL }}` should be something like `{{path/to/file|url}}` and folder should be replaced with directory in other examples. Perhaps other maintainers have a better idea how to handle this.

- Play an audio source (on repeat) forever (0 means until interrupted):

`mpg321 -l 0 {{ file(s) | URL }}`

> - Play a whole folder (recursively):

Reviewer: ```suggestion
- Play a whole directory recursively:
```

`mpg321 -B {{ folder(s) }}`

- Enable Basic Keys ( * or / - Increase or decrease volume, n - Skip song, m - Mute/unmute.) while playing:

`mpg321 -K {{ file(s) | URL }}`

- Play files randomly until interrupted:

`mpg321 -Z {{ file(s) | URL }}`

- Shuffle list of files before playing the list once:

`mpg321 -z {{ file(s) | URL }}`

> - Play all the files in the current folder(s) and subfolders, randomly (until interrupted), with Basic keys enabled:

Reviewer: ```suggestion
- Play all the files in the current directory, randomly (until interrupted), with Basic keys enabled:
```
